index.html

```diff
@@ -121,12 +121,10 @@ <h3 id="filterHeader">Filter</h3>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
     <!-- Compiled and minified JavaScript -->
     <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js"></script>
-    <script src="j.js"></script>
-
                 
     <!--JavaScript at end of body for optimized loading-->
-    <script type="text/javascript" src="js/materialize.min.js"></script>
-    <script src="j.js"></script> 
+    <!-- <script type="text/javascript" src="js/materialize.min.js"></script>  -->
```

Comment:
I don't think we need line 126 because we already have the exact same thing on line 128